specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -348,6 +348,14 @@ model Test {
   @added(APIVersions.v2024_07_01_preview)
   metricsReferenceIdentityId?: string;
 
+  @doc("Type of the managed identity built in load test engines")
+  @added(APIVersions.v2024_12_01_preview)
+  engineBuiltinIdentityType?: ManagedIdentityType = ManagedIdentityType.SystemAssigned;
+
+  @doc("Resource Ids of the managed identity built in to load test engines")
```

Comment:
Add a description, saying this is required if the `ManagedIdentityType` is UserAssigned?

Reply:
In that case should the property be engineUserIdentityIds. But given this is the case for the other Managed Identity references above. We can live with it for consistency. But we might want to update the documentation for all of the above as well?

Reply:
done